soh/soh/SaveManager.cpp

```diff
@@ -768,6 +768,10 @@ void SaveManager::LoadBaseVersion1() {
     SaveManager::Instance->LoadArray("cowsMilked", ARRAY_COUNT(gSaveContext.cowsMilked), [](size_t i) {
         SaveManager::Instance->LoadData("", gSaveContext.cowsMilked[i]);
     });
+
+    SaveManager::Instance->LoadArray("scrubsPurchased", ARRAY_COUNT(gSaveContext.scrubsPurchased), [](size_t i) {
+        SaveManager::Instance->LoadData("", gSaveContext.scrubsPurchased[i]);
+    });
```

Comment:
might need to look into how stuff works with the v1/v2 stuff here (not for this PR since it seems we have cows in both too), but it'll probably come up in the rando v2 review. maybe worth making an issue for it